src/components/GlobalPreferences/Notifications/NotificationsLogin.js

```diff
@@ -45,9 +40,12 @@ export default function NotificationsLogin({ dao, authState, onEmailChange }) {
     e => {
       const email = e.target.value
       setInputEmail(email)
-      handleEmailValidation(email)
+      if (validateEmail(email)) {
+        // Set only as valid while user typing. Use blur to set invalid
+        setEmailInvalid(false)
```

Comment:
Should we also do this validation during `handleLogin()`? If a user uses `enter` I'm not sure this would fire / change state fast enough.